src/crud.rs

```diff
@@ -32,12 +32,24 @@
 ///     convert_to_receiver_signal,
 /// );
 /// ```
+use hdk::time::sys_time;
+use hdk::prelude::ExternResult;
+
+fn now_date_time() -> ExternResult<::chrono::DateTime<::chrono::Utc>> {
+    let time = sys_time()?.as_seconds_and_nanos();
+
+    let date: ::chrono::DateTime<::chrono::Utc> =
+        ::chrono::DateTime::from_utc(::chrono::NaiveDateTime::from_timestamp(time.0, time.1), ::chrono::Utc);
+    Ok(date)
```

Comment:
You can go back to normal imports instead of the prefixed `::` versions, now that it's no longer in the body of the macro. I think it's more readable. We only do it if we have to